script/test

```diff
@@ -19,7 +19,16 @@ function do_license_check()
 
 function do_test()
 {
-    bundle_exec rake "$@"
+    echo "Running tests using the $1 Redis driver"
+
+    case $1 in
+    sync)
+        CONFIG_REDIS_ASYNC=false bundle_exec rake
+        ;;
+    async)
+        CONFIG_REDIS_ASYNC=true bundle_exec rake
+        ;;
```

Comment:
Would benefit from a catch-all case in which it blows up as the argument is unknown (as opposed to not doing anything).

Reply:
:+1: